extensions/wikia/AdEngine/js/spec/provider/gpt/adSizeFilter.spec.js

```diff
@@ -140,9 +160,10 @@ describe('ext.wikia.adEngine.provider.gpt.adSizeFilter', function () {
 	it('Filter 728x90 size for PREFOOTER_LEFT_BOXAD for small screens' +
 		' when override prefooters feature enabled', function () {
 		spyOn(mocks, 'getContentWidth').and.returnValue(1000);
+		spyOn(mocks.adSizeConverter, 'toArray').and.returnValue([[728, 90], [468, 60], [300, 250]]);
 		mocks.overridePrefootersSizes = true;
 
-		var sizesIn = [[728, 90], [468, 60], [300, 250]],
+		var sizesIn = '728x90,468x60,300x250',
```

Comment:
Let's also add one more test case for `MOBILE_TOP_LEADERBOARD` and `2x2` getting removed :japanese_ogre: